JeopardyScorekeeper/MainForm.Designer.cs

```diff
@@ -500,19 +506,103 @@ private void InitializeComponent()
             this.btnCol6Row5.BackColor = System.Drawing.Color.Blue;
             this.btnCol6Row5.Font = new System.Drawing.Font("Microsoft Sans Serif", 19.8F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
             this.btnCol6Row5.ForeColor = System.Drawing.Color.White;
-            this.btnCol6Row5.Location = new System.Drawing.Point(778, 474);
+            this.btnCol6Row5.Location = new System.Drawing.Point(778, 568);
             this.btnCol6Row5.Name = "btnCol6Row5";
             this.btnCol6Row5.Size = new System.Drawing.Size(147, 109);
             this.btnCol6Row5.TabIndex = 34;
             this.btnCol6Row5.Text = "$1000";
             this.btnCol6Row5.UseVisualStyleBackColor = false;
             this.btnCol6Row5.Click += new System.EventHandler(this.btnClicked);
             // 
+            // textCat1
+            // 
+            this.textCat1.BackColor = System.Drawing.Color.Blue;
+            this.textCat1.CharacterCasing = System.Windows.Forms.CharacterCasing.Upper;
+            this.textCat1.Font = new System.Drawing.Font("Microsoft Sans Serif", 13.8F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.textCat1.ForeColor = System.Drawing.Color.Yellow;
+            this.textCat1.Location = new System.Drawing.Point(13, 13);
+            this.textCat1.Multiline = true;
+            this.textCat1.Name = "textCat1";
+            this.textCat1.Size = new System.Drawing.Size(147, 88);
+            this.textCat1.TabIndex = 35;
+            this.textCat1.TextAlign = System.Windows.Forms.HorizontalAlignment.Center;
+            // 
+            // textCat2
+            // 
+            this.textCat2.BackColor = System.Drawing.Color.Blue;
+            this.textCat2.CharacterCasing = System.Windows.Forms.CharacterCasing.Upper;
+            this.textCat2.Font = new System.Drawing.Font("Microsoft Sans Serif", 13.8F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.textCat2.ForeColor = System.Drawing.Color.Yellow;
+            this.textCat2.Location = new System.Drawing.Point(166, 13);
```

Comment:
chance for some of these coordinates to use variables as well if they are factored out of the button coordinates above.

Reply:
Disregard for same reason as above comment.